cmd/neofs-adm/internal/modules/morph/root.go

```diff
@@ -130,7 +130,7 @@ var (
 					return fmt.Errorf("missing wallet path (use '--%s <out.json>')", storageWalletFlag)
 				}
 
-				w, err := wallet.NewWallet(storageWalletPath)
+				w, err := wallet.NewWalletFromFile(storageWalletPath)
```

Comment:
this does not save from panic if the file contains a wallet without accounts

Reply:
this PR fixed regression from the https://github.com/nspcc-dev/neofs-node/pull/2690. Before that PR, it was also done this way. but I agree, it can be improved